benchmarks/oss.py

```diff
@@ -106,8 +112,9 @@ def closure():
         mean = sum(measurements) / len(measurements)
         diff = map(lambda x: pow(x - mean, 2.0), measurements)
         std = math.sqrt(sum(diff) / (len(measurements) - 1))
-        print(f"[Regression Test] Mean: {mean:.2f} +/- {std:.2f}")
-        assert (mean - 3.0 * std) < reference_speed, "Regression detected"
+        print(f"[Regression Test] Mean speed: {mean:.2f} +/- {std:.2f}")
+        assert (mean - 3.0 * std) < reference_speed, "Speed regression detected"
+        assert max_memory < 1.05 * reference_memory, "Memory use regression detected"
```

Comment:
I added a 5% tolerance here, guessing that some CUDA or torch version changes could affect (I don't have any STD for this value, it's the max memory used over the whole run)